client/extensions/zoninator/components/settings/zone/post-card.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import SectionHeader from 'components/section-header';
+
+class PostCard extends Component {
+
+	handleRemove = () => {
+		this.props.remove( this.props.cardIdx );
+	}
+
+	handleMouseDown = ( event ) => {
+		event.stopPropagation();
+		event.preventDefault();
+	}
+
+	render() {
+		const {
+			translate,
+			post: {
+				ID,
+				URL,
+				site_ID,
+				title,
+				slug,
+			}
+		} = this.props;
+
+		const postCardClass = 'zoninator__zone__list-item';
+
+		return (
+			<SectionHeader key={ slug } label={ title } className={ postCardClass }>
```

Comment:
Uh, do we need the `key` here? Naively, we can't know that this will be rendered inside a loop -- it's normally the loop's concern to add a `key` to the component it renders (repeatedly).